### DISCUSSION TITLE
Posting my work on GitHub to share with the world

### DISCUSSION THREAD

**Discussion by DjSkooner** [community/community#133799] [New to GitHub] [answered]
How do I share my work on GitHub. I recently completed my first full stack development course and I would also like to be able to link my work with whomever might be interested. Please help

**Comment by SohanKafle** [comment, marked as answer]
Hi [DjSkooner](https://github.com/DjSkooner),

Great to hear you want to share your work on GitHub! Here's how you can do it:

<h2>Create a New Repository</h2>
1. Log in to GitHub.<br>
2. Click the "+" icon in the top right and select "New repository".<br>
3. Name your repository and click "Create repository".

<h2>Upload Your Project Files</h2>
<h3>Using Git (Command Line)</h3>
1. Open your terminal and navigate to your project directory.<br>
2. Initialize Git and add your files:

![image](https://github.com/user-attachments/assets/23bfca37-e2bf-481c-8a78-84c496d14bd7)

<h3>Using GitHub Web Interface</h3>
1. Go to your repository on GitHub.<br>
2. Click "Add file" > "Upload files".<br>
3. Drag and drop your files or select them manually.<br>
4. Click "Commit changes".

<h2>Add a README File</h2>
Create a README.md file to describe your project.

<h2>Share Your Repository</h2>
Copy and share the repository link from GitHub.
<br>
If you need any further assistance, feel free to ask!

**Reply by DjSkooner** [reply]
Thank u so much.

**Comment by DjSkooner** [comment]
Thanks a lot. I don't know where I went wrong with my GitHub account. I had to delete it and start over again and follow the instructions 
    On Sunday 28 July 2024 at 08:05:15 GMT+2, Sohan Kafle ***@***.***> wrote:  
 
 


Hi DjSkooner,

Great to hear you want to share your work on GitHub! Here's how you can do it:

Create a New Repository
1. Log in to GitHub.
2. Click the "+" icon in the top right and select "New repository".
3. Name your repository and click "Create repository".
Upload Your Project Files

Using Git (Command Line)
1. Open your terminal and navigate to your project directory.
2. Initialize Git and add your files:
image.png (view on web)

Using GitHub Web Interface
1. Go to your repository on GitHub.
2. Click "Add file" > "Upload files".
3. Drag and drop your files or select them manually.
4. Click "Commit changes".
Add a README File
Create a README.md file to describe your project.
Share Your Repository
Copy and share the repository link from GitHub.
If you need any further assistance, feel free to ask!
